metricshub-jawk-extension/src/main/java/org/sentrysoftware/metricshub/extension/jawk/JawkSourceExtension.java

```diff
@@ -64,10 +64,10 @@
 public class JawkSourceExtension implements ICompositeSourceScriptExtension {
 
 	// script to AwkTuple
-	static final Map<String, AwkTuples> awkCodeMap = new ConcurrentHashMap<>();
+	static final Map<String, AwkTuples> AWK_CODE_MAP = new ConcurrentHashMap<>();
 
 	// host to Map<String, JawkExtension>
-	static final Map<String, Map<String, JawkExtension>> extensionsMap = new ConcurrentHashMap<>();
+	static final Map<String, Map<String, JawkExtension>> EXTENSIONS_MAP = new ConcurrentHashMap<>();

```

Comment:
We should avoid default visibility here. I think it's only used in JawkSourceExtension.java. It can be private.